frontend/src/components/Topics/__tests__/Topics.spec.tsx

```diff
@@ -37,12 +37,16 @@ describe('Topics Component', () => {
 
   it('should check if the page is Topics List rendered', () => {
     setUpComponent(clusterTopicsPath(clusterName));
-    expect(screen.getByText(listContainer)).toBeInTheDocument();
+    waitFor(() => {
```

Comment:
all of the suspense stuff can be checked like this , which is recommended by Kent C Dodds 
```suggestion
expect(await screen.findByText(listContainer)).toBeInTheDocument();
```

others should be similar to this one.

Reply:
Fixed